core/src/main/java/org/web3j/protocol/ipc/UnixDomainSocket.java

```diff
@@ -59,15 +59,15 @@ public void write(String payload) throws IOException {
     @Override
     public String read() throws IOException {
         CharBuffer response = CharBuffer.allocate(bufferSize);
-        String result = "";
+        StringBuffer result = new StringBuffer();
```

Comment:
Please can you change this to use StringBuilder?

Reply:
Done, didn't that change show in the tracker?